src/snaskii.c

```diff
@@ -215,19 +215,42 @@ void showscene(char scene[][NROWS][NCOLS], int number, int menu) {
 
 /* Instantiate the snake and a set of energy blocks. */
 
+
+struct _energy_block spawn_energy_block() {
+	struct _energy_block block;
+	block.x = rand() % NCOLS;
+	block.y = rand() % NROWS;
+	return block;
+}
+
 #define BLOCK_INACTIVE -1;
 
+// Read sizeof(unsigned int) bytes from /dev/urandom
+unsigned int generate_seed() {
+	FILE *urandom = fopen("/dev/urandom", "r");
+	if (urandom == NULL) { // If we can't open /dev/urandom, use time
+		return time(NULL);
+	}
+	unsigned int seed;
+	fread(&seed, sizeof(unsigned int), 1, urandom);
```

Comment:
`snaskii.c:235:9: warning: ignoring return value of ‘fread’ declared with attribute ‘warn_unused_result’`